dotCMS/src/main/java/com/dotcms/graphql/util/GraphQLUtil.java

```diff
@@ -0,0 +1,11 @@
+package com.dotcms.graphql.util;
+
+import com.dotcms.graphql.InterfaceType;
+import java.util.Set;
+
+public class GraphQLUtil {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4926848)